mldsa/polyvec.c

```diff
@@ -16,6 +16,9 @@
 #include <string.h>
 
 #include "common.h"
+
+#if !defined(MLD_CONFIG_MULTILEVEL_NO_SHARED)
```

Comment:
I don't see how this makes sense? All the functions in this file are dependent on the K/L parameters, so in multi-level build you CANNOT get away with building this file only once.
This guard should be removed.

Reply:
done